src/scythe-bidder/create-room.tsx

```diff
@@ -4,29 +4,34 @@ import React from "react";
 import { jsx } from "@emotion/core";
 import { Button, Card, notification, Select, Form } from "antd";
 import client from "./client";
-import { MAX_PLAYERS, MIN_PLAYERS, SCYTHE_BIDDER } from "./constants";
+import { MIN_PLAYERS, SCYTHE_BASE, SCYTHE_IFA, MAX_PLAYERS_BASE, MAX_PLAYERS_IFA } from "./constants";
 
 export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
   const [numPlayers, setNumPlayers] = React.useState(2);
+  const [gameType, setGameType] = React.useState<string>(SCYTHE_BASE);
+  const [maxPlayers, setMaxPlayers] = React.useState(MAX_PLAYERS_BASE);
```

Comment:
Could we derive this piece of state from the value of gameType (i.e. `const maxPlayers = gameType === SCYTHE_BASE ? MAX_PLAYERS : MAX_PLAYERS_IFA`? Best to keep React state as minimal as possible to prevent bugs.

Reply:
I didn't know it could be done this way :-). Done.